packages/frontend/src/components/modules/CalculationForm/CalculationForm.module.scss

```
@@ -0,0 +1,533 @@
+// .calculation-owner {
+//   margin: 0 auto 320px;
+
+//   @include bp-1919 {
+//     margin: 0 auto 300px;
+//   }
+
+//   @include bp-1059 {
+//     width: 100%;
+//     margin: 0 auto 220px;
+//   }
+
+//   @include bp-929 {
+//     margin: 0 auto 216px;
+//   }
+
+//   @include bp-699 {
+//     margin: 0 auto 174px;
+//   }
+
+.heading {
+  @include  headline3;
+  width: 883px;
+  margin: 0 auto 40px;
+  color: $gray-blue-500;
+  text-align: center;
+
+  @include bp-929 {
+    @include headline-regular-24;
+    width: 87%;
+    margin: 0 auto 20px;
+  }
+
+  &_block {
+    display: block;
+
+    @include bp-699 {
+      display: inline;
+    }
+  }
+}
+
+.text {
+  @include headline4;
+  width: 883px;
+  margin: 0 auto 40px;
+  color: $iris-400;
+  text-align: center;
+
+  @include bp-929 {
+    @include headline5;
+    width: 87%;
+    margin: 0 auto 30px;
+  }
+
+  @include bp-699 {
+    margin: 0 auto 20px;
+  }
+}
+
+.form {
+  position: relative;
+  padding: 40px 42px;
+  background-color: $iris-100;
+  border-radius: 30px;
+
+  @include bp-1059 {
+    padding: 35px 42px 40px;
+    border-radius: 0;
+  }
+
+  @include bp-699 {
+    padding: 28px 16px 100px;
+  }
+}
+
+.list,
+.list-rooms {
+  display: flex;
+  justify-content: space-between;
+}
+
+.list {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

Тут кстати можно вот так написать:
```suggestion
  grid-template-columns: repeat(4, 1fr);
```